packages/terra-application-layout/tests/wdio/application-menu-spec.js

```diff
@@ -12,8 +12,8 @@ describe('ApplicationMenu', () => {
     Terra.should.themeCombinationOfCustomProperties({
       testName: 'themed',
       properties: {
-        '--terra-application-menu-background-color': 'pink',
-        '--terra-application-menu-header-background-color': 'blue',
+        '--terra-application-layout-menu-background-color': 'pink',
+        '--terra-application-layout-menu-header-background-color': 'blue',
```

Comment:
Just curious, know why 9ish screenshots were created for a test change that should have changed 2 screenshots?

Reply:
So it turned out some screenshots got kept around after a large group of merge conflicts from before Christmas. I already went through and deleted the ones we weren't supposed to keep. Gonna double check other packages to see if they got some hangers on themselves as well.